### ISSUE TITLE
Typeschema does not return all information

### ISSUE THREAD

**Issue by cbos** [eclipse-emfcloud/emfcloud-modelserver#40]
We can request a typeschema via a request to this url: api/v1/typeschema?modeluri=...

But the returned schema, only returns information about the direct class, but not information of the super class.
We have a test model where we have a common ecore model with interfaces, and we have 2 derived ecore models. The specific field attributes or relations defined in the derived model are returned, but the attibutes and relations in the common model are not returned even no references.
Due to that the returned typeschema is incomplete. 

Either it should return a reference to these schemas or it should include it.

**Comment by ndoschek** [comment]
Hi! Sorry for the late reply. You are right, thanks for raising that issue.
Personally I would prefer to add the elements from the common model instead of references in regards to usage, i.e. one request by the consumer is then enough to get the complete type information.
If you have already a solution in mind we would happily accept a contribution for this. Otherwise we will take a look at it in the next weeks.

**Comment by cbos** [comment]
@ndoschek 
We did not make a solution yet. For now we created the typeschemas manually to explore the possibilities of jsonforms. 

If we start working on it we will definitely keep in touch to see how we can contribute it, but we did not plan for that yet.

In case there is a working solution committed by one of you, we can easily update our setup and test it to see if that works based on what we experience right now to give the feedback as soon as possible.
